src/tab.group.ts

```diff
@@ -189,7 +189,8 @@ export default class TabGroup extends LitElement {
       this.#tabElements.includes(clickedTab)
     ) {
       this.#selectedTab = clickedTab;
-      this.#updateTabsPanelsAndTabBar();
+      this.#updateSelectedTabAndPanel();
+      this.#updateSelectedTabIndicator();
```

Comment:
Separated `#updateTabsPanelsAndTabBar()` into `#updateSelectedTabAndPanel()` and `#updateSelectedTabIndicator()` so I can call only what I need to (`#updateSelectedTabIndicator()`) in ` #onTabListResize()`.

Calling `#updateTabsPanelsAndTabBar()` in `#onTabListResize()` here also caused the `changes tabs on keyboard interaction` test to fail because it changes the selected tab `tabIndex` in the next tick. Happy to explain in more detail if you like.

Additionally, and for whatever it's worth, Tab Group, Tab, and Tab Panel are queued up for a rework, which they sorely need. They're the last of our early components that we've yet to rewrite.